Sources/SwiftGen/Config/ConfigEntry.swift

```diff
@@ -84,14 +78,24 @@ extension Config.Entry {
     }
     return typedValue
   }
+
+  static func parseValueOrArray<T, U>(yaml: Any, key: String? = nil, parseValue: (T) throws -> U) throws -> [U] {
+    if let value = yaml as? T {
+      return [try parseValue(value)]
+    } else if let value = yaml as? [T] {
+      return try value.map { try parseValue($0) }
+    } else {
+      throw Config.Error.wrongType(key: key, expected: "\(T.self) or Array of \(T.self)", got: type(of: yaml))
```

Comment:
I know I'm the one who wrote that code in a previous command, but I'm not sure how the error will be displayed in the console when we use `(T.self)` in string interpolation, does it read nicely? I didn't actually test it